LondonBritishLibrary/orient/BLorient772.xml

```diff
@@ -124,13 +91,104 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                                             <citedRange unit="page">220a</citedRange>
                                         </bibl>
                                     </listBibl>
-                                    
+
                                 </source>
                             </recordHist>
                         </adminInfo>
                     </additional>
+
+                    <msPart xml:id="p1">
+                                  <msIdentifier></msIdentifier>
+                                  <msContents>
+                                  <msItem xml:id="p1_i1">
+                                    <locus target="#2"></locus>
+                                    <title type="complete" ref="LIT1957Mashaf"/>
+
+                    </msItem>
+                    </msContents>
+                    <physDesc>
+                        <objectDesc form="Leaf">
+                            <supportDesc>
+                                <support>
+                                    <material key="parchment"/>
+                                </support>
+                                <extent>
+                                    <measure unit="leaf">1</measure>
+                                </extent>
+                            </supportDesc>
+
+                        </objectDesc>
+
+                    </physDesc>
```

Comment:
The ownership note on fol. 2r belongs to this msPart. And tere should be a handNote, for it was definitely written by another hand as part of another ms

Reply:
Sorry I can't see the handnote and an addition, have you added them?